tests/test_logme.py

```diff
@@ -0,0 +1,79 @@
+"""
+Test functions for the LogME metric (src/locomoset/metrics/logme.py)
+"""
+
+import numpy as np
+import pytest
+
+from locomoset.metrics.logme import logme
+
+
+def test_logme_n_samples_catch(dummy_features_perfect, dummy_labels):
+    """Test that the LogME metric catches that the sample size isn't great enough to
+    compute a consistent score
+    """
+    with pytest.raises(AssertionError):
+        logme(dummy_features_perfect, dummy_labels)
+
+
+def test_logme_perfect_features(
+    dummy_features_perfect, dummy_labels, logme_minimum_datatpoints_required
+):
+    """Test that LogME returns a value greater than zero for perfect features, with less
+    than zero only seen for random features.
+
+    Empirically at least 3500 datapoints are required for a representative score."""
```

Comment:
See other comments re 3500, probably just remove this line in the docstring for now.

Reply:
Done